### DISCUSSION TITLE
Indeksihäiriö 12.9.

### DISCUSSION THREAD

**Discussion by johannaraisa** [KohaSuomi/Koha#1407] [Huolto- ja häiriötiedotteet]
Indeksit ovat jotenkin korruptoituneet, syynä taitaa olla levyjärjestelmäremontti. Palautamme 11.9. indeksin ja päivitämme sen ajan tasalle.

**Comment by emta001** [comment]
Sekä Täti että Vaara on palautettu ja indeksit ovat päivittymässä.

**Comment by emta001** [comment]
Lumme ja Kyyti palautettu ja päivittymässä.

**Comment by emta001** [comment]
Siili ja OUTI palautettu ja päivittymässä.

**Comment by emta001** [comment]
Kirkes palautettu ja päivittymässä.

**Comment by emta001** [comment]
Myös Helle, Lappi ja Vaski on nyt saatu palautettua ja laitettua päivittymään.

**Comment by emta001** [comment]
OUTIn ja Vaskin päivityksissä on vielä jotain vialla. Odotellaan, että muut päivitykset valmistuvat ja yritetään sitten näiden kanssa uudestaan.

**Reply by emta001** [reply]
Samaa vikaa myös Lapissa.

**Comment by emta001** [comment]
Lapin, OUTIn ja Vaskin indeksien varmuuskopiot ovat puutteelliset ja ne joudutaan  valitettavasti uudelleenindeksoimaan.

**Reply by emta001** [reply]
Lapissa on uudelleenindeksointi käynnissä. Jotta palvelin ei kuormittuisi liiaksi, odottelemme että muut päivitykset rupeavat valmistumaan ennen kuin OUTIn ja Vaskin uudelleenindeksointi käynnistetään.

**Reply by johannaraisa** [reply]
Laitetaan OUTI ja Vaski vasta illalla, tämä päivä menee vajaalla indeksillä.

**Comment by emta001** [comment]
Muiden kimppojen, paitsi Lapin, OUTIn ja Vaskin, indeksit on nyt päivitetty.
